### ISSUE TITLE
File is corrupted 

### ISSUE THREAD

**Issue by Jenarou** [LumaTeam/Luma3DS#844]
Hi,  the luma8.1.7z file is corrupted.

Thanks.

**Comment by Shadow322** [comment]
Then redownload it thanks

**Comment by Jenarou** [comment]
I am following the guide for 3ds hacking and I downloaded all files except this one,   the file is corrupted no matter how many times i download it .... try to download it and see yourself.

**Comment by ghost** [comment]
File is not corrupted, tested.

**Comment by ihaveamac** [comment]
both 8.1 and 8.1.1 downloads verified here.

**Comment by Jenarou** [comment]
ihaveamac , Hikari-chin thank you 
this is what i exactly get when i try to open the file, note its the only 7z file I download that do this.

anyway thanks guys ill try to download it on another OS and see how it goes maybe its my system.

cheers many thanks for the help.

Edit: I also tried to ignore the message and continue anyway, the file doesn't work, I found another one though and my problem solved, I just want you to know about the issue if there is one.
![screen shot 2017-09-12 at 4 18 36 pm](https://user-images.githubusercontent.com/31848869/30327843-2d22c80e-97d6-11e7-9730-dccb92edf9d6.png)
